sdk/core/Azure.Core.Testing/src/Azure.Core.Testing.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We are trying to keep the directories under the "service", "core" in this case represent packages that we ship as there will be tooling in place that ensure it has all the things that we have for shipping the package. For that reason we have been putting test libraries and utilities under the primary tests folder that defines/needs them.  Could you do the same for this? I'd suggest putting them under `sdk/core/Azure.Core/tests/TestFramework/Azure.Core.TestFramework.csproj` this is what we have done for KeyVault fro example https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Microsoft.Azure.KeyVault/tests/TestFramework.

Reply:
Sounds good, I'll open a follow-up PR.

Reply:
> For that reason we have been putting test libraries and utilities under the primary tests folder that defines/needs them

This library is intended to be used by all track 2 client SDK tests, is `sdk/core/Azure.Core/tests/TestFramework` still the best place for it?

Reply:
I think that is reasonable given they will also all depend on Azure.Core and that will be the main dependency in this test framework library.

Reply:
Feels slightly strange to have a project inside another projects folder.

Reply:
Yes it is a little weird but I'd like to continue with this strategy until we decide it doesn't work well. I'm definitely not opposed to potentially creating a "Common" folder at some point but I'm trying to wait to see what patterns emerge first.